accessman.go

```diff
@@ -611,8 +611,12 @@ func (a *accessMan) addPeerAccess(remotePub *btcec.PublicKey,
 
 // removePeerAccess removes the peer's access from the maps. This should be
 // called when the peer has been disconnected.
-func (a *accessMan) removePeerAccess(peerPubKey string) {
-	ctx := btclog.WithCtx(context.TODO(), "peer", peerPubKey)
+func (a *accessMan) removePeerAccess(peerPub *btcec.PublicKey) {
```

Comment:
I think it's unnecessary to pass the full struct while only the str is used - we could either,
- use `btclog.Fmt("peer", "%x", peerPubKey))` here in the ctx, which prints the key in hex.
- construct the `ctx` outside the method and pass it to the `removePeerAccess`.